README.md

```diff
@@ -109,7 +109,15 @@ module.exports = function(environment) {
       whitelistUrls: [],
 
       /**
-       * Options to pass directly to Raven.js. Note: whitelistUrls and
+       * Raven.js option.
+       *
+       * @type {Array}
+       * @default []
+       */
+      ignoreUrls: [],
```

Comment:
I'm not sure this will work. 🤔  The option that I added in https://github.com/damiencaselli/ember-cli-sentry/pull/85 is on the service itself, not on the serialized config object. The reason for this is that it can support regular expressions and serializing and deserializing them in the config object correctly is unnecessary overhead.

Reply:
Ah, I hadn't thought of regepx. So the correct location for this option is on the service itself?

Reply:
> So the correct location for this option is on the service itself?

yep

Reply:
I've been fighting with trying to set ignoreUrls and ignoreErrors in config/environment.js.  As well, i've been trying to put regex's on there.

What is the proper way to configure these, as of how?  hardcode them on the service itself/don't use the service provided by the addon?

http://damiencaselli.github.io/ember-cli-sentry/ certainly isn't conclusive as to what to do.